internal/provider/gce/environ_firewall.go

```diff
@@ -24,6 +25,11 @@ func (env *environ) globalFirewallName() string {
 	return common.EnvFullName(env.uuid)
 }
 
+// modelFirewallName returns the name to use for model level firewall.
+func (env *environ) modelFirewallName() string {
```

Comment:
We have a method globalFirewallName() already which returns the same result I wonder if we should use that.

Reply:
Maybe I shouldn’t create an explicit function for this just to replicate the code.
Convinced, let me change this